test/unit/adapter.test.js

```diff
@@ -31,6 +31,17 @@ describe('MongooseAdapter', () => {
     assert.isFalse(adapter.isFiltered());
   });
 
+  it('Should properly instantiate multiple adapters', async () => {
+    const adapter1 = new MongooseAdapter('mongodb://localhost:27001/casbin', { ...MONGOOSE_OPTIONS, autoCreate: true });
+    const adapter2 = new MongooseAdapter('mongodb://localhost:27001/casbin2', { ...MONGOOSE_OPTIONS, autoCreate: true });
+
+    assert.instanceOf(adapter1, MongooseAdapter);
+    assert.isFalse(adapter1.isFiltered());
+
+    assert.instanceOf(adapter2, MongooseAdapter);
+    assert.isFalse(adapter2.isFiltered());
+  });
+
```

Comment:
I tried to add a test here, but I noticed some others were failing, so not sure if I did this the right way.

If you can direct me, I'd be happy to implement proper tests.